inception-kb/src/main/java/de/tudarmstadt/ukp/inception/kb/graph/RdfUtils.java

```diff
@@ -161,13 +163,14 @@ protected RepositoryException convert(Exception aE)
                 .escapeString(language) + "\")).";
         }
         String QUERY = String.join("\n",
-            InferencerVariableStore.PREFIX_OWL,
-            InferencerVariableStore.PREFIX_RDF,
-            InferencerVariableStore.PREFIX_RDFS,
+                "PREFIX rdfs: <" + RDFS.NAMESPACE + ">",
```

Comment:
Please comment what this Query does

Reply:
DONE

Reply:
@Rentier Tests are already added in the other branch.